src/parkapi_sources/converters/bahn_v2/mapper.py

```diff
@@ -26,6 +33,13 @@ def map_static_parking_site(bahn_input: BahnParkingSiteInput) -> StaticParkingSi
             has_realtime_data=False,  # TODO: change this as soon as Bahn offers proper rate limits
             static_data_updated_at=datetime.now(tz=timezone.utc),
             public_url=bahn_input.url,
+            purpose=next(
```

Comment:
Why looping here at all? I mean, you select the type `BahnParkingSiteCapacityType.PARKING` anyway?